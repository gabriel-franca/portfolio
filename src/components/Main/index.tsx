import Logo from '~/assets/logo.svg'

const Main = () => (
  <div>
    <Logo aria-label="Gabriel FranÃ§a" />
  </div>
)

export default Main
