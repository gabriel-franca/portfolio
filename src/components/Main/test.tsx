import { render, screen } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('should render the main', () => {
    const { container } = render(<Main />)

    expect(screen.getByLabelText(/Gabriel Fran√ßa/i)).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
