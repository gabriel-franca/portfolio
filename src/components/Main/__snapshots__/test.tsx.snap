// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the main 1`] = `
<div>
  <svg
    aria-label="Gabriel FranÃ§a"
    fill="currentColor"
    viewBox="0 0 100 115"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M75.002 71.76l-25 14.356-25-14.356V43.05l25-14.355 25 14.355L100 28.71 50.002 0 0 28.71v57.422l50.002 28.71L100 86.132V57.405L75 43.05 50.115 57.495 75.002 71.76"
    />
  </svg>
</div>
`
